build.sbt

```diff
@@ -75,23 +75,26 @@ lazy val parsing = project("akka-parsing")
   .addAkkaModuleDependency("akka-actor")
 
 lazy val httpCore = project("akka-http-core")
+  .enablePlugins(Unidoc)
   .settings(Dependencies.httpCore)
-  .settings(Version.versionSettings)
+  .settings(VersionGenerator.versionSettings)
   .dependsOn(parsing)
   .addAkkaModuleDependency("akka-stream")
   .addAkkaModuleDependency("akka-stream-testkit", "test")
 
 lazy val http = project("akka-http")
+  .enablePlugins(Unidoc)
   .dependsOn(httpCore)
 
 lazy val http2Support = project("akka-http2-support")
-  .enablePlugins(JavaAgent)
+  .enablePlugins(JavaAgent, Unidoc)
   .disablePlugins(MimaPlugin) // experimental module still
   .settings(javaAgents += Dependencies.Compile.Test.alpnAgent)
   .dependsOn(httpCore, httpTestkit % "test", httpCore % "test->test")
   .addAkkaModuleDependency("akka-stream-testkit", "test")
 
 lazy val httpTestkit = project("akka-http-testkit")
+  .enablePlugins(Unidoc)
```

Comment:
Is this required when generating Javadoc with genjavadoc?

Reply:
I see now. When `akka.genjavadoc.enabled=true` then `Unidoc` requires `GenJavadocPlugin` and this enables it.

I think we should rename `Unidoc` plugin as it brings some confusion here. The sbt-unidoc plugin works at the root project. So enabling something called `Unidoc` on sub-projects is confusing. What about `BootstrapGenjavadoc`?

Reply:
I have done it like so here: https://github.com/akka/akka/pull/23850/commits/8d597966866c8ea24c2340c47b1029f1426039bd